_posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Welcoming OLS Resident Fellow Seun Olufemi to OLS-9 Delivery Team
+authors:
+- malvikasharan
+- seunolufemi123
+image: https://images.unsplash.com/photo-1554457606-ed16c39db884
```

Comment:
@seunolufemi123 Are you happy with this image or would you prefer [this](https://unsplash.com/photos/white-ceramic-teapot-on-white-wooden-box-OGND72jS-HE) or something else?